modules/react/tabs/lib/TabsList.tsx

```diff
@@ -43,33 +44,29 @@ export const useTabsList = composeHooks(
   useListResetCursorOnBlur
 );
 
-const StyledStack = styled(Stack)<StyledType>({
-  '::after': {
-    content: '""',
-    position: 'sticky',
-    height: 52,
-    minWidth: 30,
-    background: `linear-gradient(to right,rgba(255,255,255,0),white);`,
-    zIndex: 1,
-    right: 0,
-    top: 0,
-    pointerEvents: 'none',
-  },
-});
+const StyledStack = styled(Stack)<StyledType>(({maskImage}) => ({
```

Comment:
If we do stick with this approach of applying `maskImage` to `StyledStack`, I would limit the `maskImage` typing to the `StyledStack` in TabsList rather than applying it to `StyledType` in the components.ts file (which isn't specific to Tabs and is used all over the place):

```suggestion
const StyledStack = styled(Stack)<StyledType & {maskImage?: string}>(({maskImage}) => ({
```

That said, to @NicholasBoll's point in his [earlier comment](https://github.com/Workday/canvas-kit/pull/1705#discussion_r937905533):

> We may want to make this more complicated to support a fade effect only when there is something to be scrolled. This should reduce the likelihood that the last tab is active and faded which looks a bit silly.

I do wonder if this is worth looking into given the odd visual effect Nicholas is referring to of the last active tab being faded:

![last-active-tab-fade](https://user-images.githubusercontent.com/146020/182974825-308ee28a-d040-404a-8168-6107952545f7.jpg)

Reply:
Hi @anishatulai! Checking in to see if there's been any updates to this PR based on my last comments above. Do you have any leads on how to address the fadeout effect on the last tab?

Reply:
No worries, this has been a trickier issue than anticipated. Thanks for the update!